__tests__/sagas-test.ts

```diff
@@ -0,0 +1,18 @@
+import { runSaga } from 'redux-saga';
+import { onGetCounters } from '../src/sagas';
+import { getCounters, getCountersSuccess } from '../src/actions';
+import fetchMock from 'fetch-mock';
+
+it('will fetch counters - success', async () => {
+  const expectedCounters = [{ count: 6 }, { count: 4 }, { count: 2 }, { count: 1 }];
+  fetchMock.mock('https://api.myjson.com/bins/ywmke', expectedCounters);
+
+  const dispatched: any[] = [];
+  const result = await runSaga({
+    dispatch: (action) => { dispatched.push(action) },
+  }, onGetCounters, getCounters()).toPromise();
```

Comment:
I'm guessing that `.toPromise()` is the new `.done` in the new version of Redux-saga?